src/JWT/JwtDecoder.cs

```diff
@@ -249,9 +248,9 @@ public void Validate(JwtParts jwt, params byte[][] keys)
             {
                 _jwtValidator.Validate(decodedPayload, asymmAlg, bytesToSign, decodedSignature);
             }
-            else
+            else if (algorithm is ISymmetricAlgorithm symmAlg)

```

Comment:
Should we use pattern matching `switch/case` here?

Reply:
fixed